dbt_semantic_interfaces/parsing/schemas.py

```diff
@@ -269,6 +269,28 @@
 }
 
 
+saved_query_schema = {
+    "$id": "saved_query_schema",
+    "type": "object",
+    "properties": {
+        "name": {"type": "string"},
+        "description": {"type": "string"},
+        "metrics": {
+            "type": "array",
+            "items": {"type": "string"},
+        },
+        "group_bys": {
+            "type": "array",
+            "items": {"type": "string"},
+        },
+        "where": {
+            "type": "array",
+            "items": {"type": "string"},
+        },
+    },
+    "additionalProperties": False,
```

Comment:
Nit: I don't think we have to, but we should probably mark the properties `name` and `metrics` to be required. I think parsing will error out either way if its not specified, as the creation of the object will fail. But specifying them in the jsonschema spec will give better errors I believe.

Reply:
Updated.